coconut/const.py

```diff
@@ -0,0 +1,232 @@
+    "tkinter.simpledialog": ("SimpleDialog", (3,)),
+    "tkinter.tix": ("Tix", (3,)),
+    "tkinter.ttk": ("ttk", (3,)),
+    "tkinter.constants": ("Tkconstants", (3,)),
+    "tkinter.dnd": ("Tkdnd", (3,)),
+    "tkinter.colorchooser": ("tkColorChooser", (3,)),
+    "tkinter.commondialog": ("tkCommonDialog", (3,)),
+    "tkinter.filedialog": ("tkFileDialog", (3,)),
+    "tkinter.font": ("tkFont", (3,)),
+    "tkinter.messagebox": ("tkMessageBox", (3,)),
+    "tkinter.simpledialog": ("tkSimpleDialog", (3,)),
+    "urllib.robotparser": ("robotparser", (3,)),
+    "xmlrpc.client": ("xmlrpclib", (3,)),
+    "xmlrpc.server": ("SimpleXMLRPCServer", (3,)),
+    "urllib.request": ("urllib2", (3,)),
+    "urllib.parse": ("urllib2", (3,)),
+    "urllib.error": ("urllib2", (3,)),
+    "io.StringIO": ("StringIO", (3,)),
+    "io.BytesIO": ("BytesIO", (3,)),
+    "collections.abc": ("collections", (3, 3))
+}
+
+default_whitespace_chars = " \t\f\v"
+
+# end: CONSTANTS
+
+#-----------------------------------------------------------------------------------------------------------------------
+# COMMAND CONSTANTS:
+#-----------------------------------------------------------------------------------------------------------------------
+
+code_exts = [".coco", ".coc", ".coconut"] # in order of preference
+comp_ext = ".py"
+
+main_sig = "Coconut: "
+debug_sig = ""
+
+default_prompt = ">>> "
+default_moreprompt = "    "
+
+watch_interval = .05 # seconds
+
+color_codes = { # unix/ansii color codes, underscores in names removed
+    "bold": 1,
+    "dim": 2,
+    "underlined": 4,
+    "blink": 5,
+    "reverse": 7,
+    "default": 39,
+    "black": 30,
+    "red": 31,
+    "yellow": 33,
+    "blue": 34,
+    "magenta": 35,
+    "cyan": 36,
+    "lightgray": 37,
+    "darkgray": 90,
+    "lightred": 91,
+    "lightgreen": 92,
+    "lightyellow": 93,
+    "lightblue": 94,
+    "lightmagenta": 95,
+    "lightcyan": 96,
+    "white": 97,
+    "defaultbackground": 49,
+    "blackbackground": 40,
+    "redbackground": 41,
+    "greenbackground": 42,
+    "yellowbackground": 43,
+    "bluebackground": 44,
+    "magentabackground": 45,
+    "cyanbackground": 46,
+    "lightgraybackground": 47,
+    "darkgraybackground": 100,
+    "lightredbackground": 101,
+    "lightgreenbackground": 102,
+    "lightyellowbackground": 103,
+    "lightbluebackground": 104,
+    "lightmagentabackground": 105,
+    "lightcyanbackground": 106,
+    "whitebackground": 107
+    }
+end_color_code = 0
+
+version_long = "Version " + VERSION_STR + " running on Python " + " ".join(sys.version.splitlines())
+version_banner = "Coconut " + VERSION_STR
+if DEVELOP:
+    version_tag = "develop"
+else:
+    version_tag = VERSION_TAG
+tutorial_url = "http://coconut.readthedocs.org/en/" + version_tag + "/HELP.html"
+documentation_url = "http://coconut.readthedocs.org/en/" + version_tag + "/DOCS.html"
+
+icoconut_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "icoconut")
```

Comment:
I think this should work, since `const.py` is at the same level as `command.py` was, but you have to be very careful when working with `__file__`.